cumulusci/core/source_transforms/tests/test_transforms.py

```diff
@@ -528,3 +530,129 @@ def test_source_transform_parsing__bad_transform():
         )
 
         assert "destroy_the_things is not valid" in str(e)
+
+
+def test_strip_unwanted_files():
```

Comment:
This is a really nice integration test. Well done. The only thing I'd suggest (but this is not a blocker) is a comment explaining how the setup exercises all of the use cases you're validating here.

Reply:
Added comments below